source/core/NumericString.cpp

```diff
@@ -19,6 +19,7 @@ SDG
 #include "ExecutionContext.h"
 #include "StringUtilities.h"
 #include "TDCodecVia.h"
+#include "../../ThirdParty/utfcpp/source/utf8.h"
```

Comment:
I think it would be better to add the top level directory to the compile search path, so you can just do 
#include "ThirdParty/utfcpp/source/utf8.h".

But fine with submitting as-is for now.

Reply:
Good idea. Will do that later.